lib/sqlalchemy/sql/selectable.py

```diff
@@ -6813,6 +6815,24 @@ def intersect_all(
         """
         return CompoundSelect._create_intersect_all(self, *other)
 
+    def into(
+        self,
+        target: Union[str, TableClause, Table],
+        *,
+        schema: Optional[str] = None,
+        temporary: bool = False,
+        if_not_exists: bool = False,
+    ) -> "CreateTableAs":
+        from .ddl import CreateTableAs
```

Comment:
if this ddl import can't be at the top., we have a system used for in-method imports that's threadsafe

```
@util.preload_module("sqlalchemy.ddl")
def my_method(self, ...):
    ddl = util.preloaded.ddl
    return ddl.CreateTableAs(...)
```